.pipelines/pipeline.user.linux.yml

```diff
@@ -47,4 +47,7 @@ package:
       repository_name: 'cdpxlinux'                        # only supported ones are cdpx acr repos
       tag: 'cidev'                                        # OPTIONAL: Defaults to latest. The tag for the built image. Final tag will be 1.0.0alpha, 1.0.0-timestamp-commitID.
       latest: false                                       # OPTIONAL: Defaults to false. If tag is not set to latest and this flag is set, then tag as latest as well and push latest as well.
+      publish_unique_tag: true                            # If set, the image in the registry is tagged with the unique tag generated by CDPx 
+      metadata_file:
+        artifact_path: 'linux-image-meta.json'            # If defined, the drop outputs relative path to the file into which JSON metadata about the created image is emitted.
```

Comment:
you will have to validate linux-image-meta.json & windows-image-meta.json files exists. if it doesnt exist, fail the release.